zkcloud/chain.json

```diff
@@ -0,0 +1,114 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "zkcloud",
+  "status": "live",
+  "network_type": "testnet",
```

Comment:
Testnets belong in the /testnets/ directory. The chainname must also have 'testnet', e.g., "zkcloudtestnet" (as well as the corresponding directory name)